app/static/js/index.js

```diff
@@ -2,380 +2,550 @@
+						addRow(nameWrapper, "school", "School", "university", person);
+					}
+					addRow(nameWrapper, "year", "Graduation Year", "graduation-cap", person, null, true, true);
+					// Don't show student Phone#
+					if (person.school_code !== "YC") {
+						addRow(nameWrapper, "phone", "Phone Number", "phone", person, person.phone ? "tel:" + person.phone : null, false);
+					}
+					addRow(nameWrapper, "website", "Website", "globe", person, person.website, false, true);
+
+					// Append the name wrapper to the header wrapper
+					headerWrapper.appendChild(nameWrapper);
+
+					// Append the header wrapper to the person container
+					personContainer.appendChild(headerWrapper);
+
+					// Create a container for the metadata pills
+					let pills = document.createElement("div");
+					pills.className = "pills";
+
+					// Add NETid pills
+					if (person.netid || person.upi) {
+						if (person.netid) {
+							let pill = document.createElement("div");
+							pill.className = "pill";
+							pill.textContent = "NetID " + person.netid;
+							pills.appendChild(pill);
+						}
+						if (person.upi) {
+							let pill = document.createElement("div");
+							pill.className = "pill";
+							pill.textContent = "UPI " + person.upi;
+							pills.appendChild(pill);
+						}
+					}
+
+					// Add metadata pills
+					addPill(pills, "pronouns", "Pronouns", "comments", person);
+					addPill(pills, "title", "Title", "tags", person);
+					addPill(pills, "major", "Major", "book", person);
+					addPill(pills, "birthday", "Birthday", "birthday-cake", person);
+					addPill(pills, "address", "Address", "home", person);
+
+					// Append the pills container to the person container
+					personContainer.appendChild(pills);
+
+					// Append the person container to the main list
+					p.list.appendChild(personContainer);
+				}
+
+				// Hide the 'loading' element once the data is fetched and processed
+				p.loading.style.display = "none";
+			});
+	}
 }
 
 runSearch();
 
-window.onscroll = function(e) {
-    if (2 * window.innerHeight + window.scrollY >= document.body.offsetHeight) {
-        loadNextPage();
-        // Temporarily set so that we won't load tons of pages at once
-        pagesFinished = true;
-    }
-
-    p.scrollTop.classList.toggle('shown', window.scrollY > (2 * window.innerHeight));
+window.onscroll = function (e) {
+	if (2 * window.innerHeight + window.scrollY >= document.body.offsetHeight) {
+		loadNextPage();
+		// Temporarily set so that we won't load tons of pages at once
+		pagesFinished = true;
+	}
+
+	p.scrollTop.classList.toggle(
+		"shown",
+		window.scrollY > 2 * window.innerHeight
+	);
 };
+
+let filters = document.querySelectorAll("#filters > div");
+filters.forEach((filter, index) => {
+	const IMPORTANT_FILTERS = 4; // NUMBER OF DEFAULT FILTERS
+	if (index >= IMPORTANT_FILTERS) {
+		filter.style.display = "none";
+	}
+});
+
+document
+	.getElementById("expand_filters")
```

Comment:
Declare element aliases in the `p` object at the top of the file.
Also I don't like having "clear filters" and "advanced" both have the same icon, can you change up the icons perhaps?